File: cpp/program25.cpp
// program10.c

#include<iostream>
using namespace std;

class Print
{
    public:
        int iNo;

    Print(int X)
    {
        iNo = X;
    }

    void Display()
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= iNo; iCnt++)
        {
            cout<<"Fuck You...!!!"<<"\n";
        }
    }
};

int main()
{
    int iValue = 0;

    cout<<"Enter Frequency "<<'\n';
    cin>>iValue;

    Print pobj(iValue);
    pobj.Display();

    return 0;
}

File: java/program25.java
import java.util.*;

class program25
{
    public static void Display(int no)
    {
        int iCnt = 0;

        for(iCnt = 1; iCnt <= no; iCnt++)
        {
            System.out.println("Fuck you bitch...");
        }
    }
    public static void main(String arg[])
    {
        int iValue = 0;
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number");
        iValue = sobj.nextInt();

        Display(iValue);
    }    
}